### ISSUE TITLE
Not all focused inputs are properly indicated

### ISSUE THREAD

**Issue by GVodyanov** [nextcloud/calendar#7931]
### Steps to reproduce

Issue:
- When opening editfull and selecting the event title, it is shown like this:

<img width="782" height="85" alt="Image" src="https://github.com/user-attachments/assets/740dc716-4dd6-4d18-a79b-0901155c94d1" />

Other inputs might also be affected

### Expected behavior

It should have a thick border

### Actual behavior

Thin border

### Web server

None

### Database engine version

None

### PHP engine version

None

### Updated from an older installed version or fresh install

None

### List of activated apps

```Text

```

### Nextcloud configuration

```Text

```

### Web server error log

```Text

```

### Log file

```Text

```

### Browser log

```Text

```
